server/views/pages/appointments/partials/occurrence-movement-slip.njk

```diff
@@ -14,8 +14,7 @@
 
             <dl class='movement-slip__summary'>
                 {{ labelAndValue('Appointment', '<ul class="govuk-list">' +
```

Comment:
Strictly speaking, this doesn't have to be an unordered list anymore as it only has one row